.github/workflows/build.yaml

```diff
@@ -61,3 +63,15 @@ jobs:
         type=sha,format=short,event=branch
         type=raw,value=latest-ci
         type=raw,value=latest-ci_{{date 'X' }}
+  build_embedded_element_call:
+    # Use the embedded package vite build
+    uses: ./.github/workflows/build-element-call.yaml
+    with:
+      package: embedded
+      vite_app_version: ${{ github.event.release.tag_name || github.sha }}
+    secrets:
+      SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
```

Comment:
Aren't these already in the nuild-element-call.yaml?

Reply:
It does feel a bit convoluted to me, but I think `build-element-call.yaml` is intended to be a generic build step.

Then, the secrets are treated as a parameterised input to the generic build step.

So, depending on where `build-element-call.yaml` is called from you could pass different secrets in.

In reality we don't use this.